library/ssl_ciphersuites.c

```diff
@@ -2064,6 +2064,19 @@ int mbedtls_ssl_get_ciphersuite_id( const char *ciphersuite_name )
     return( cur->id );
 }
 
+size_t mbedtls_ssl_ciphersuite_get_cipher_key_bitlen( const mbedtls_ssl_ciphersuite_t *info )
+{
+#if defined(MBEDTLS_CIPHER_C)
+    const mbedtls_cipher_info_t * const cipher_info =
+      mbedtls_cipher_info_from_type( info->cipher );
+
+    return( mbedtls_cipher_info_get_key_bitlen( cipher_info ) );
+#else
```

Comment:
That's a reasonable approach. The function that would be needed to compute the key size from the `info->cipher` without using `mbedtls_cipher_` functions has not yet landed development, though it happens to [exist in a parallel PR](https://github.com/ARMmbed/mbedtls/pull/5434/files#diff-b9b23ef0e168bb80241e10e0308a5e7859922a01c0090984b484f2e9f388ce92R1278).

We can do a follow-up once both this PR and #5434 are merged to fill the `#else` part.